File: Day_23/Q91.java
import java.util.*;

class Q91{
    static Scanner sc = new Scanner(System.in);

    //Function To Check Anagram
    static boolean check_anagram(String str1,String str2){

        //Length Check
        if(str1.length() != str2.length()) return false;

        char arr1[] = str1.toCharArray();
        char arr2[] = str2.toCharArray();

        Arrays.sort(arr1);
        Arrays.sort(arr2);

        //Compare Both Strings
        return Arrays.equals(arr1,arr2);
    }

    public static void main(String args[]){
        String str1 = sc.nextLine();
        String str2 = sc.nextLine();

        //Print Result
        if(check_anagram(str1,str2)) System.out.println("Anagram Strings");
        else System.out.println("Not Anagram Strings");

        sc.close();
    }
}

File: Day_23/Q91.cpp
#include<bits/stdc++.h>
using namespace std;

//Function To Check Anagram
bool check_anagram(string str1,string str2){

    //Length Check
    if(str1.length() != str2.length()) return false;

    sort(str1.begin(),str1.end());
    sort(str2.begin(),str2.end());

    //Compare Both Strings
    return str1 == str2;
}

int main(){
    string str1,str2;

    getline(cin,str1);
    getline(cin,str2);

    //Print Result
    if(check_anagram(str1,str2)) cout << "Anagram Strings";
    else cout << "Not Anagram Strings";

    return 0;
}
